civcraftConfigs/gimmelconfig.yml

```diff
@@ -153,16 +153,159 @@ monster:
      blocks_to_spawn_in:
       - WATER
       - STATIONARY_WATER
+   dirtyspider:
+     identifier: dirtyspider
+     type: CAVE_SPIDER
+     name: Dirty Spider
+     on_hit_message: Cant Catch Me
+     spawn_chance: 0.01
+     drops:
+      String:
+       material: STRING 
+       amount: 16
+       chance: 1
+      spidereyes:
+       material: SPIDER_EYE 
+       amount: 8
+       chance: 1
+      fermspidereyes:
+       material: FERMENTED_SPIDER_EYE  
+       amount: 4
+       chance: 0.3
+     buffs:
+      Fast2:
+       type: SPEED
+       level: 2
+     on_hit_debuffs:
+      poison:
+       type: POISON
+       level: 1
+       duration: 10s
+       chance: 1.0 
+      blinded:
+       type: BLINDNESS
+       level: 1
+       duration: 5s
+       chance: 1.0 
+      Slowness:
+       type: SLOW
+       level: 2
+       duration: 5s
+       chance: 1.0 
+     blocks_to_spawn_on:
+      - STONE
+      - DIRT
+      - GRASS
+      - STATIONARY_LAVA
+      - STATIONARY_WATER 
+     blocks_to_spawn_in:
+      - AIR
+      - STATIONARY_LAVA
+      - STATIONARY_WATER 
+     health: 100
+     despawn_on_chunk_unload: true
+     can_pickup_items: false
+     y_spawn_range: 16
+   darkzombiearcher:
+     identifier: undeadarcher
+     type: ZOMBIE
+     name: Dark Archer
+     on_hit_message:  "**Zombie** **Snarls** **Crunch** **Rawr**"
+     spawn_chance: 0.05
+     drops:
+      Flesh:
+       material: ROTTEN_FLESH
+       amount: 8
+       chance: 1
+      zquiver:
+       material: ARROW
+       amount: 8
+       chance: 1
+       lore: Zombie Quiver
+      pistonwithstickyblood:
+       material: PISTON_STICKY_BASE
+       amount: 2
+     equipment:
+      ZombieArrows:
+       material: BOW 
+       enchants:
+        Punch4:
+         enchant: ARROW_KNOCKBACK
+         level: 4
+        Fire1:
+         enchant: ARROW_FIRE
+         level: 1
+      chainchest2:
+       material: CHAINMAIL_CHESTPLATE
+       enchants:
+        Prot2:
+         enchant: PROTECTION_ENVIRONMENTAL
+         level: 2
+      chainlegs2:
+       material: CHAINMAIL_LEGGINGS
+       enchants:
+        Prot2:
+         enchant: PROTECTION_ENVIRONMENTAL
+         level: 2
+      chainboots2:
+       material: CHAINMAIL_BOOTS
+       enchants:
+        Prot2:
+         enchant: PROTECTION_ENVIRONMENTAL
+         level: 2
+      skullhelm2:
+       material: SKULL
```

Comment:
ahh i see, in the future will it be able to use other items for a helm? i know theres a plugin called mythic mobs i use for my server and it does almost the same thing as EE but you can do soo much more to the mobs, all the ideas for these mobs can from that plugin and with that you could have mob wear just about anyhting including glass blocks on the head that is colored aswell as color the armor they wear